doc/rados/operations/health-checks.rst

```diff
@@ -493,7 +493,7 @@ This warning can be disabled with::
 BLUESTORE_NO_PER_POOL_OMAP
 __________________________
 
-Starting with the Octopus release, BlueStore tracks omap space utilization
+Starting with the Pacific release, BlueStore tracks omap space utilization
```

Comment:
It seems to me that per-pool omap were introduced in Octopus by https://github.com/ceph/ceph/pull/29292
Have I missed something?

Reply:
Then I'm not sure how to explain /a/sage-2021-03-23_02:00:05-upgrade:octopus-x-wip-sage-testing-2021-03-22-1729-distro-basic-gibba/5989342